develop/webservices/WS_v1/WS_About_v1/Using_the_Web_Services_v1.md

```diff
@@ -4,22 +4,63 @@ uid: Using_the_Web_Services_v1
-   > -  If two-step authentication is required, use *ConnectAppAndInfo* (see [ConnectAppAndInfo](xref:ConnectAppAndInfo)) and *ConnectAppAndInfoStep2* (see [ConnectAppAndInfoStep2](xref:ConnectAppAndInfoStep2)) instead of *ConnectApp*.
-   > -  If the connection is not used for 5 minutes, the session will end. You will then need to connect again to request a new connection ID.
+Most methods require a connection string, which is a token identifier received upon authentication. To authenticate, either use the [ConnectApp](xref:ConnectApp) method, or, if two-step authentication is required, use [ConnectAppAndInfo](xref:ConnectAppAndInfo) and [ConnectAppAndInfoStep2](xref:ConnectAppAndInfoStep2).
 
-1. Call any of the other methods. Remember to pass along the connection ID you received in step 2.
+When you authenticate, make sure to pass along the name and version of the client application as provided by Skyline Communications as part of the DataMiner software license.
 
-   For a list of available methods, see [Methods (v1)](xref:WS_Methods_v1_overview#methods-v1).
+> [!NOTE]
+> If the connection is not used for 5 minutes, the session will end. You will then need to connect again to request a new connection ID.
 
-1. Call the *LogOut* method to log out. See [LogOut](xref:LogOut).
+## Interfaces and WSDL files
 
-> [!WARNING]
-> We strongly recommend that you use HTTPS when accessing the Web Service APIs over public Internet. If you do not do so, all information – including logon credentials – will be sent over the Internet as plain, unencrypted text.
+The DataMiner Web Services (v1) are accessible via the following interfaces.
+
+| Protocol | Version | Address          |
+|----------|---------|------------------|
+| SOAP     | v1      | `http://DmaNameOrIpAddress/API/v1/soap.asmx` |
+| JSON     | v1      | `http://DmaNameOrIpAddress/API/v1/json.asmx` |
+
+> [!NOTE]
+> For security reasons, by default, it is not possible to test web API requests from an external machine using HTTP POST instead of SOAP. To enable this, uncomment the following section in the file *web.config* in the folder *C:\\Skyline DataMiner\\Webpages\\API*:<br><br> \<!-- uncomment to test webservices outside localhost:<br> \<protocols><br> \<add name="HttpGet"/><br> \<add name="HttpPost"/><br> \</protocols><br> //-->
+
+The WSDL files can be found on the following addresses.
+
+| Protocol | Version | Address                |
+|----------|---------|------------------------|
+| SOAP     | v1      | `http://DmaNameOrIpAddress/API/v1/soap.asmx?WSDL` |
+| JSON     | v1      | `http://DmaNameOrIpAddress/API/v1/json.asmx?WSDL` |
+
+## SOAP
+
+When SOAP requests are used, the URL should have the structure `http(s)://{DMA IP}/api/v1/soap.asmx`, regardless of which method is chosen.
+
+Note that the fields of each method need to be present within the method's name enclosing tag.
```

Comment:
This isn't entirely clear to me. Do you mean that you always need to include all input fields of a method?

Reply:
I mean that the methods fields such as: connection, dmaID and elementID need to be within the GetElement tag as demonstrated below:

        <GetElement xmlns="http://www.skyline.be/api/v1">
            <connection>string</connection>
            <dmaID>int</dmaID>
            <elementID>int</elementID>
        </GetElement>